### PR TITLE
Buttons 'main menu' and 'draw graph' are added. User status added. Us…

### PR DESCRIPTION
**Pull request by RaEzhov**
…er function to drawing a graph is always converted to lower case.

### CODE REVIEW

File: source/core/chat_bot_template.py [hunk 1]
```diff
@@ -17,14 +17,31 @@
 
 logger = logging.getLogger(__name__)
 
+MAIN, GRAPH = range(2)
```
Comment (Korostast) [inline source/core/chat_bot_template.py:20]: Use Python class Enum to introduce constant state variables

File: source/core/chat_bot_template.py [hunk 2]
```diff
@@ -34,7 +51,16 @@ def chat_help(update: Update, context: CallbackContext):
 
 def echo(update: Update, context: CallbackContext):
     """Echo the user message."""
```
Comment (Korostast) [inline source/core/chat_bot_template.py:53]: I suppose, it is not just echo. Give morу details on how the function processes user input. Maybe you should rename the function and change docstring

File: source/core/chat_bot_template.py [hunk 3]
```diff
@@ -17,14 +17,31 @@
 
 logger = logging.getLogger(__name__)
 
+MAIN, GRAPH = range(2)
 
+chats_status_dict = {}
 # Define a few command handlers. These usually take the two arguments update and
 # context. Error handlers also receive the raised TelegramError object in error.
```
Comment (Korostast) [inline source/core/chat_bot_template.py:24]: Maybe remove this comment? I don't understand what it references to

**Review by Korostast — APPROVED** [review]
Nice, bot becomes more comfortable to use

File: source/core/chat_bot_template.py (hunk 1 above)
Comment (RaEzhov) [reply source/core/chat_bot_template.py:20]: Okay, I'll fix it in next commit.

File: source/core/chat_bot_template.py (hunk 2 above)
Comment (RaEzhov) [reply source/core/chat_bot_template.py:53]:
```suggestion
    """Check user status and directs his message to suitable function."""
```

File: source/core/chat_bot_template.py (hunk 3 above)
Comment (RaEzhov) [reply source/core/chat_bot_template.py:24]: I agree with it.
```suggestion
```

**Pull request merged**